src/layouts/AppBar/AutoCompleteSearch.component.tsx

```
@@ -0,0 +1,116 @@
+                                {inputValue && (
+                                    <IconButton
+                                        size="small"
+                                        onClick={handleClear}
+                                        sx={{ padding: 0 }}
+                                        aria-label="Clear input"
+                                    >
+                                        <IconContainer
+                                            size={20}
+                                            sx={{ color: 'text.secondary' }}
+                                        >
+                                            <ClearIcon></ClearIcon>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)